package.json

```diff
@@ -3,15 +3,15 @@
 	"version": "1.0.0",
 	"private": true,
```

Comment:
Is that intentional given
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private
and
https://www.npmjs.com/package/samsung-tv-remote